input/blog/2018-01-17-cake-v0.25.0-released.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Cake v0.25.0 released
+category: Release Notes
+author: devlead
+---
+
+Version 0.25.0 of Cake has been released.
+
+The first release of 2018 is here! It contains mostly improvements and bug fixes, but it also has one potential breaking change. 
+
+The most notable change in this release is that we no longer use the NuGet.exe CLI tool when installing addins, tools, modules and scripts form NuGet sources in our [preprocessor directives](/docs/fundamentals/preprocessor-directives).
```

Comment:
I'm guessing this should say "from" not "form"?

Reply:
yes, I'll fix.